deploy/role.yaml

```diff
@@ -1,13 +1,14 @@
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
```

Comment:
Not sure about this, but I thinking RBAC is generated by operator-sdk, isn't it? I recall there were annotations on controller specifying what RBAC rules it requires.

Reply:
I have follow the operator-sdk guide to convert into [cluster-scoped one](https://github.com/operator-framework/operator-sdk/blob/master/doc/operator-scope.md)